### ISSUE TITLE
Problems deploying new image

### ISSUE THREAD

**Issue by D063520** [ec-doris/EuKnowledgeGraph#75]
I tried to redeploy the image, but I get:
<img width="1728" alt="Screenshot 2022-04-28 at 20 36 01" src="https://user-images.githubusercontent.com/10883585/165823068-bc18010c-cf6f-4e57-ae73-e397a7835647.png">
and it never happens

**Comment by raphdom** [comment]
This is a problem of permissions, now I've changed and grant you access to push into this repo. Can you try again please?

**Comment by D063520** [comment]
Same problem ...
<img width="1728" alt="Screenshot 2022-04-29 at 09 18 59" src="https://user-images.githubusercontent.com/10883585/165900644-f983ceb3-55ce-4de7-b2ce-dce05005a14d.png">

**Comment by ecdoris** [comment]
Don't understand, can you try again in another terminal and certify that you're using the dennis user for cnect-prime account, normally you have 2 users, one for cnect-prime and another for cnect-production.

**Comment by madewild** [comment]
Aaaah, a talking fish! :P

**Comment by D063520** [comment]
That was it! I pushed the image now, I do not remember the workflow, it should redeploy automatically or not?

**Comment by raphdom** [comment]
no, if you change the tag you need to change the .yaml file and update with the new tag, otherwise you just need to delete the pod and it will be created with the new version of the image from the registry

**Comment by D063520** [comment]
So I should change it here:

https://github.com/ec-doris/gitops-linkedopendata/blob/main/environments/development/base/apps/wikibase/base/wikibase-bundle.yaml

right? 

But how do I know the name of the new image?

I got this when pushing

7993b86d9aee: Layer already exists 
5d6a6c05080c: Layer already exists 
b258e9d25731: Layer already exists 
ddd4adc58b9e: Layer already exists 
ae6d225e2209: Layer already exists 
9321ff862abb: Layer already exists 
1.35.5-wmde.3: digest: sha256:cca0f78c2fea26b2c6f3c16d65d854d2dada1595792ecdee6c37f9fdd00400cb size: 9087

**Comment by raphdom** [comment]
I am not changing the name of the tag, I think we could leave like this and change if mediawiki change the version

**Comment by D063520** [comment]
I'm confused .... so what should I do to see the image I just pushed redeployed? Or is it already. I guess no because I do not see the changes ....
